docs/resources/worker.md

```diff
@@ -15,12 +15,9 @@ The resource allows you to create a self-managed worker object.
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- `scope_id` (String) The scope for the worker.
-
 ### Optional
 
+- `scope_id` (String) The scope for the worker.
```

Comment:
Should we document what happens if this is not set? Does it get created in the global scope?

Reply:
Curious if a `scope_id` is optional, my understanding was that it was always global? If that is the case shouldn't we just deprecate this field

Reply:
Right now it defaults to global, though there has been discussion about allowing workers to exist at other scopes so I'm not sure if we should deprecate it?

Reply:
Na I think leave it in place if it is something we might be supporting in the future. Another option can be to set a default value of global via the provider